### PR TITLE
Highway/ramp lane bifurcation

### PR DESCRIPTION
**Pull request by dgearhart**
# Issue

Added logic to detect and mark the highway/ramp lane bifurcation as a fork.

## Example#1
image
image

``` diff
< BEFORE
< 1: Drive west on I 75 South.
> AFTER
> 1: Drive west on I 75 South.
> 2: Keep left to stay on I 75 South toward Dayton.
>    2: TURN_LANES: [ *slight_left* ACTIVE | *slight_left*;through ACTIVE | through ] ACTIVE_TURN_LANES
```

---

## Example#2
image
image
``` diff
< BEFORE
< 1: Drive east on A 96.
> AFTER
> 1: Drive east on A 96.
> 2: Keep left to stay on A 96 toward München-Zentrum/Salzburg/Garmisch-Partenkirchen.
>    2: TURN_LANES: [ *through* ACTIVE | *through* ACTIVE | slight_right | slight_right ] ACTIVE_TURN_LANES
```
---

## Example#3
image
image

``` diff
< BEFORE
< 1: Drive west on I 696 West/Walter P Reuther Freeway.
> AFTER
> 1: Drive west on I 696 West/Walter P Reuther Freeway.
> 2: Keep left to stay on I 696 West/Walter P Reuther Freeway toward I 96 West/Lansing.
>    2: TURN_LANES: [ *slight_left* ACTIVE | *slight_left* ACTIVE | slight_right | slight_right ] ACTIVE_TURN_LANES
```

---

## Tasklist

 - [x] Add tests
 - [x] Add #fixes with the issue number that this PR addresses
 - [x] Update the docs with any new request parameters or changes to behavior described
 - [x] Update the [changelog](CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
    * FIXED: Missing locking in incident handler needed to hang out to scop lock rather than let the temporary disolve [#3046](https://github.com/valhalla/valhalla/pull/3046)
    * FIXED: Continuous lane guidance fix [#3054](https://github.com/valhalla/valhalla/pull/3054)
    * FIXED: Fix reclassification for "shorter" ferries and rail ferries (for Chunnel routing issues) [#3038](https://github.com/valhalla/valhalla/pull/3038)
+   * FIXED: Highway/ramp lane bifurcation [#3088](https://github.com/valhalla/valhalla/pull/3088)
+
```
Comment (gknisely) [inline CHANGELOG.md:13]: nit.  extra space

**Comment by gknisely** [comment]
@dgearhart what happens when the lane count is not set

**Comment by dgearhart** [comment]
> @dgearhart what happens when the lane count is not set

@gknisely the lane_count would set to 1 and result would be the same as before. This update is dependent on lane_count since there is not another attribute to make the decision

**Review by gknisely — APPROVED** [review]

**Comment by dgearhart** [comment]
Fixes https://github.com/mapbox/nav-api-intake/issues/636

**Pull request merged**